velox/common/memory/tests/MemoryArbitratorTest.cpp

```diff
@@ -69,6 +71,44 @@ TEST_F(MemoryArbitrationTest, create) {
   }
 }
 
+TEST_F(MemoryArbitrationTest, queryMemoryCapacity) {
+  {
+    // Reserved memory is not enforced when no arbitrator is provided.
+    auto allocator = std::make_shared<MallocAllocator>(8L << 20);
+    MemoryManager manager{
+        {.capacity = (int64_t)allocator->capacity(),
+         .queryMemoryCapacity = 4L << 20,
```

Comment:
Consider to have a warning in memory manager if queryMemoryCapacity != kMaxMemory but arbitrator is not set? Thanks!